wire-ios/Wire-iOS/Sources/UserInterface/Settings/ChangingDetails/ChangeEmailViewModel.swift

```diff
@@ -0,0 +1,52 @@
+//
+// Wire
+// Copyright (C) 2024 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import WireSyncEngine
+
+final class ChangeEmailViewModel {
+
+    private(set) var state: ChangeEmailState
```

Comment:
I wonder if `ChangeEmailState` and `ChangeEmailViewModel ` should be merged.
Is there a valid reason for the state type to exist?
Everything here (in the view model) seems to depend on the state.

Reply:
Done: https://github.com/wireapp/wire-ios/commit/96668abcfc15f538e83b01f1885d59fdc4c757a8